src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Ball {
+    private Colors color;
+    private int number;
+
+    public void setColor(Colors color) {
+        this.color = color;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public Colors getColor() {
+        return color;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+}

```

Comment:
Your Ball class should look like this:

```java
public class Ball {
    private Colors color;
    private int number;

    public Ball(Colors color, int number) {
        this.color = color;
        this.number = number;
    }

    @Override
    public String toString() {
        return "The ball color is " + color + " and has a number " + number;
    }
}
```

Reply:
Nadpisałem metodę toString. Wygląda na to, że działa. Przeoczyłem ten wymóg w instrukcji. Ale przyznam się, że nie rozumiem, czemu jest to lepsze rozwiązanie. Nie rozumiem też czemu mam usunąć gettery i settery jednocześnie utrzymując prywatny modyfikator dostępu przy zmiennych.

Reply:
Pease use English to communicate.

Getters can stay, but once the Ball is created in the lottery, we don't want to allow to make changes to this ball, therefore do not add setters to the `Ball` class.

Reply:
Because if you don't override the toString() method then instead of a message like you created, you'll get an insignificant message about the object's reference